app/models/miq_provision_virt_workflow.rb

```diff
@@ -722,31 +722,7 @@ def allowed_organizational_units(options = {})
       _log.info("sysprep_domain_name=<#{ou_domain}> does not match previous=<#{@last_ou_domain}> - recomputing")
       @last_ou_domain = ou_domain
       @ldap_ous = {}
-      details   = MiqProvision.get_domain_details(ou_domain, true, @requester)
-      return @ldap_ous if details.nil?
-
-      options[:host]      = details[:ldap_host]  if details.key?(:ldap_host)
-      options[:port]      = details[:ldap_port]  if details.key?(:ldap_port)
-      options[:basedn]    = details[:base_dn]    if details.key?(:base_dn)
-      options[:user_type] = details[:user_type]  if details.key?(:user_type)
-      l = MiqLdap.new(options)
-      userid   = details[:bind_dn]
-      password = details[:bind_password]
-      if userid.nil? || password.nil?
-        _log.info("LDAP Bind with Defaults")
-        ldap_bind = l.bind_with_default
-      else
-        _log.info("LDAP Bind with userid=<#{userid}>")
-        ldap_bind = l.bind(userid, password)
-      end
-
-      if ldap_bind == true
-        ous = l.get_organizationalunits
-        _log.info("LDAP OUs returned: #{ous.inspect}")
-        ous.each { |ou| @ldap_ous[ou[0].dup] = ou[1].dup } if ous.kind_of?(Array)
-      else
-        _log.warn("LDAP Bind failed")
-      end
+      return @ldap_ous
```

Comment:
@tinaafitz It does not make sense to keep any of the internals of this method.  The method should just return an empty hash so it does not raise an error if called from an existing dialog.

```
def allowed_organizational_units(options = {})
  {}
end
```

Later we can put together a separate PR with a data migration to remove all the deprecated dialog fields and methods from the dialogs.